CICE/CMakeLists.txt

```diff
@@ -127,13 +127,15 @@ target_sources(OM3_cice PRIVATE
   CICE/cicecore/drivers/nuopc/cmeps/CICE_InitMod.F90
   CICE/cicecore/drivers/nuopc/cmeps/CICE_RunMod.F90
   CICE/cicecore/drivers/nuopc/cmeps/cice_wrapper_mod.F90
-  CICE/cicecore/drivers/nuopc/cmeps/ice_comp_nuopc.F90
   CICE/cicecore/drivers/nuopc/cmeps/ice_import_export.F90
   CICE/cicecore/drivers/nuopc/cmeps/ice_prescribed_mod.F90
   CICE/cicecore/drivers/nuopc/cmeps/ice_scam.F90
   CICE/cicecore/drivers/nuopc/cmeps/ice_shr_methods.F90
 )
+
 add_patched_source(OM3_cice CICE/cicecore/drivers/nuopc/cmeps/ice_mesh_mod.F90)
+add_patched_source(OM3_cice CICE/cicecore/drivers/nuopc/cmeps/ice_comp_nuopc.F90)

```

Comment:
@micaeljtoliveira - is there a neater syntax than this for adding multiple patched sources?

Reply:
No, not really. If you have many files to patch then it might be worth putting them in a list and loop over the list contents. Something like this:

```
list(APPEND FilesToPatch CICE/cicecore/drivers/nuopc/cmeps/ice_mesh_mod.F90 CICE/cicecore/drivers/nuopc/cmeps/ice_comp_nuopc.F90)
foreach(FILE IN LISTS FilesToPatch)
  add_patched_source(OM3_cice ${FILE})
endforeach()
```

Another option would be to add a variant of `add_patched_source`  that takes multiple files:
```cmake
add_patched_source(OM3_cice CICE/cicecore/drivers/nuopc/cmeps/ice_mesh_mod.F90 CICE/cicecore/drivers/nuopc/cmeps/ice_comp_nuopc.F90)
```

IMO, none of them looks much neater than what you have here, but happy to hear other opinions.

Reply:
Ok thanks - ill leave it alone :)